builds/jazz-build-module/lambda-events-module.groovy

```diff
@@ -0,0 +1,175 @@
+#!groovy?
```

Comment:
lets rename this to aws-lambda..... as we will have more files specific to other cloud platforms at some point.

Reply:
done